schemas/common.yaml

```diff
@@ -0,0 +1,73 @@
+responses:
+  schemas:
+
+    AppointmentFetchRequest:
+      description: Successfull -- It will return Tasks array
+      content:
+       application/json:
+        schema:
+         type: array
+         items:
+          $ref: 'appointment.yaml#/components/schemas/Appointment'
+            
+    AppointmentGuid:
+     description:  Successfull --> It will return id of task created
+     content:
+       application/json:
+        schema:
+         type: object
+         properties:
+           id:
+            $ref: 'appointment.yaml#/components/schemas/Appointment/properties/id'
+         additionalProperties: false
+      
+
+    BadReqError:
+     description: Bad Request Error
+     content:
+       application/json:
+        schema:
+         type: object
+         properties:
+           message:
+            type: string
+           errorcode:
+            type: boolean
+            example: false
+         additionalProperties: false
+         
+    AppointmentError:
+     description: Returns a Conflict message there is already an appointment at specified time interval
+     content:
+           application/json:
+            schema:
+             title: CreateContentProviderPermissionDTO
+             required:
+              - title
+              - startTime
+              - endtime
+             type: object
+             properties:
+              title:
+                $ref: '#schemas/appointment/components/schemas/Appointment/properties/title'
+              startTime:
+                $ref: '#schemas/appointment/components/schemas/Appointment/properties/startTime'
+              endtime:
+                $ref: '#schemas/appointment/components/schemas/Appointment/properties/endTime'
+            additionalProperties: false
+
+
+    InternalServerError:
+      description: The server encountered an unexpected condition that prevented it from fulfilling the request
+      content:
+        application/json:
+         schema:
+          type: object
+          properties:
+            error:
+             description: The error occured due to an internal server issue
+             type: string
+             example: Error occured on server side
+          additionalProperties: false
```

Comment:
`BadReqError`, `AppointmentError` and `InternalServerError` these are not schemas. it will placed under responses